keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/VersionedArtifactContainer.kt

```diff
@@ -0,0 +1,13 @@
+package com.netflix.spinnaker.keel.api
+
+import com.netflix.spinnaker.keel.api.artifacts.DeliveryArtifact
+
+/**
+ * Interface implemented by [ResourceSpec] types that contain versioned artifacts, typically compute resources.
+ */
+interface VersionedArtifactContainer {
```

Comment:
I moved this from `keel-core` and added the `Container` suffix for clarity as we were tripping up on the old name.